localnet/Makefile

```diff
@@ -0,0 +1,7 @@
+all:
+	docker build --no-cache --tag localnet/node ../.. -f evmos/Dockerfile --build-arg commit_hash=$(shell bash -c 'read -p "Version or Commit Hash: " version; echo $$version') --build-arg extra_flags=$(shell bash -c 'read -p "Extra flags: " flags; echo $$flags') --build-arg USERNAME=$$USER
+
+# TODO: add repo flag 
```

Comment:
What about this TODO?

Reply:
Updated to prompt the user to select the repository